Gemfile

```diff
@@ -13,11 +13,7 @@ end
 gem 'rails', '~> 5.0.1'
 gem 'rails-controller-testing'
 
-# Add CacheFile backend module.
-# Ref: https://github.com/svenfuchs/i18n/pull/423
-# Support numeric keys in Simple backend.
-# Ref: https://github.com/svenfuchs/i18n/pull/422
-gem 'i18n', github: 'wjordan/i18n', branch: 'cdo'
+gem 'i18n', '1.3.0'
```

Comment:
(forgot to mention) thanks for updating this back to the latest published gem! :+1: :tada: